sbndcode/BlipRecoSBND/Utils/BlipUtils.cc

```diff
@@ -196,15 +200,17 @@ namespace BlipUtils {
         // check that the times are similar (we don't want to merge
         // together a blip that happened much later but in the same spot)
         if( fabs(blip_i.Time - blip_j.Time) > 5 ) continue;
-        float d = (blip_i.Position-blip_j.Position).Mag();
+        float d = TMath::Sqrt((blip_i.Position-blip_j.Position).Mag2());
```

Comment:
You can directly use `.R()`:

```suggestion
        float d = (blip_i.Position-blip_j.Position).R();
```
Also note that for determining the closest point, you don't really need to take the square root of the square distance, because the minimum distance is also the minimum distance square, so you can look for the latter and save a tiny bit of computing time.
An awkward feature of the new vectors is that they have `Mag2()` and `R()`, but not `Mag()` and `R2()`. 🤷